src/AssertWpErrorTypeSpecifyingExtension.php

```diff
@@ -38,7 +38,12 @@ public function specifyTypes(MethodReflection $methodReflection, MethodCall $nod
     {
         $args = $node->getArgs();
 
-        return $this->typeSpecifier->create($args[0]->value, new ObjectType('WP_Error'), TypeSpecifierContext::createTruthy());
+        return $this->typeSpecifier->create(
+            $args[0]->value,
+            new ObjectType(\WP_Error::class),
```

Comment:
Or is `ObjectType('WP_Error')` preferred?

Reply:
Less strings -> brighter future 🌻